packages/utils/src/v2/TableId.ts

```diff
@@ -35,6 +35,7 @@ export class TableId {
     // assumes tableId is a 32-byte hex string, otherwise it left-pads with zeros (for numbers)
     // this is scary, since zero padding is different depending on the type (bytes types vs number types)
     // TODO: fix this after we move tableIds to bytes32 instead of uint256
+    // TODO: check if this is still relevant
     const tableIdBytes = new Uint8Array(32);
     tableIdBytes.set(tableId.slice().reverse());
     tableIdBytes.reverse();
```

Comment:
@holic can you help me figure out how to change this logic to make it work with `tableIds` that are `bytes32`? Do we assume `tableId` to be `uint256` anywhere else in the client code?

Reply:
I think we can just change this to
```
const tableIdBytes = new Uint8Array(32);
tableIdBytes.set(tableId);
```
which ensures right-zero padding. But if ethers is already passing bytes32 values as zero padded hex, then we won't even need this.

(before it would reverse, then right-pad, then reverse again, so that the result is left-padded)

We could also use the viem `pad` helpers for this to make it more clear what's going on.